docs/dashboard/configuration/plugins.md

```diff
@@ -66,8 +64,24 @@ Go to _Configuration_ -> _Plugins_ -> _Adyen_ and fill in the fields:
 
 - **Notification user:** In addition to checking request signatures, you can confirm received notification by checking the provided user in the notification. Provide a user in the _Authentication_ section of your Adyen customer area and copy it here. See [Adyen docs](https://docs.adyen.com/development-resources/webhooks#set-up-notifications-in-your-customer-area).
 
-- **Notification password:** In addition to checking request signatures, you can confirm received notification by checking the provided user’s password in the notification . Provide a password in the _Authentication_ section of your Adyen customer area and copy it here. See [`Adyen docs`](https://docs.adyen.com/development-resources/webhooks#set-up-notifications-in-your-customer-area).
+- **Notification password:** In addition to checking request signatures, you can confirm received notification by checking the provided user’s password in the notification . Provide a password in the _Authentication_ section of your Adyen customer area and copy it here. See [Adyen docs](https://docs.adyen.com/development-resources/webhooks#set-up-notifications-in-your-customer-area).
+
+- **Enable native 3d secure:** Saleor uses 3D Secure redirect authentication by default. If you want to use native 3D Secure authentication, enable this option. For more details see Adyen documentation: [native-3ds2](https://docs.adyen.com/checkout/3d-secure/native-3ds2), [redirect-3ds2-3ds1](https://docs.adyen.com/checkout/3d-secure/redirect-3ds2-3ds1)
```

Comment:
It's either "3-D Secure" (the official name) or "3D Secure". Please use the proper name here and in the actual UI.